core/vm/gas_table.go

```diff
@@ -88,10 +87,23 @@ func memoryCopierGas(stackpos int) gasFunc {
 	}
 }
 
-func gasExtCodeSize(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
+func makeGasPush(pushCount uint) gasFunc {
+	return func(evm *EVM, contract *Contract, stack *Stack, mem *Memory, pc uint64, memorySize uint64) (uint64, error) {
+		var constantGas uint64 = 3 // TODO don't hardcode this
```

Comment:
note: can remove this because the constant portion is already covered by the EVM.